### ISSUE TITLE
igx-grid multi-cell selection behavior

### ISSUE THREAD

**Issue by rssureshkumar** [IgniteUI/igniteui-angular#5823]
**Description**
Feature request or Suggestion required to achieve the following  using  "igniteui-angular@7.3.x"

1.Option to disable columns from selection or allow to select multiple cells in only one column.
2. Option to deselect selected cell using Ctrl + mouse click

**Steps to reproduce**
Please consider the example for the explanation 
https://stackblitz.com/edit/angular-prevent-cell-selection-svtvvt

1. I am dragging from the first row of "Reorder Level" column and dragging down to few columns and moving to "Action" column before releasing mouse now it selects both "Reorder Level" column and "Action" column. As per requirement user not allowed to select cells from "Action" column. 

2. Ctrl + mouse click allows selecting multiple cells. Same way While Ctrl key pressed and the user clicks on the selected cell it should deselect the selected cell.

**Comment by zdrawku** [comment]
@rssureshkumar, below are my comments based on your questions:

> 1. Option to disable columns from selection or allow to select multiple cells in only one column.

We are currently working on Column selection functionality that will expose the `selectable` column input, by using it you can make a column selectable or now. Check the latest [alpha version](https://github.com/IgniteUI/igniteui-angular/releases/tag/9.1.0-alpha.4) for more information.

> 2. Option to deselect selected cell using Ctrl + mouse click

Our cell selection is excel-based and we try to follow the selection approach defined by Microsoft excel. On `Ctrl + left mouse click` the cell won't be deselected - check out the [Multi-cell selection specification](https://github.com/IgniteUI/igniteui-angular/wiki/Grid-Multi-cell-selection-Specification) for more information. 

In conclusion, thank you for your questions and recommendations, the first point will be available with the upcoming 9.1.0 release, meanwhile, you can use the alpha version that we release on a weekly basis. As for the second point, I had a discussion with @radomirchev about this, and the current behavior is by design and we do not plan to change that in the future.
